nevergrad/benchmark/xpbase.py

```diff
@@ -57,15 +59,20 @@ def __init__(
         self.optimizer = optimizer
         self.budget = budget
         self.num_workers = num_workers
-        self.executor = execution.MockedTimedExecutor(batch_mode)
+        ng_cpus_per_task = int(os.getenv("NG_CPUS_PER_TASK", "1"))
```

Comment:
This is only for benchmark so let's make it clear
```suggestion
        ng_cpus_per_task = int(os.getenv("NG_BENCHMARK_CPUS_PER_TASK", "1"))
```